tests/integration/test.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+import os
+import shutil
+
+lichen_installation_dir = "/usr/local/submitty/Lichen"
+lichen_test_playground = "/usr/local/submitty/Lichen/test_output"
+
+
+class TestLichen(unittest.TestCase):
+    def setUp(self):
+        shutil.rmtree(os.path.join(lichen_test_playground, 'test_lichen'), ignore_errors=True)
+        os.makedirs(os.path.join(lichen_test_playground, 'test_lichen'))
+
+    def tearDown(self):
+        shutil.rmtree(os.path.join(lichen_test_playground, 'test_lichen'))
+
+    def testLichen(self):
+        self.maxDiff = None
+        for test_case in sorted(os.listdir('../data/test_lichen')):
+            print(f"running integration test for {test_case}...")
+            # make the fake directory where the config.json is saved
+            base_path = f"{lichen_test_playground}/test_lichen/{test_case}"
+            os.makedirs(base_path)
+
+            # copy the input files from /data to the the new path
+            data_path = f"{os.getcwd()}/../data/test_lichen/{test_case}/input"
+            shutil.copyfile(f"{data_path}/config.json", f"{base_path}/config.json")
+
+            # run Lichen
+            os.system(f"bash {lichen_installation_dir}/bin/process_all.sh {base_path} {data_path}")
```

Comment:
Don't use `os.system` as it swallows the error code, as well as outputting all of stdout which we probably don't want.

Better to use [`subprocess.check_call`](https://docs.python.org/3/library/subprocess.html#subprocess.check_call) and pipe out the `stderr` and ignore the `stdout`.